roles/config-docker-machine/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Compiled and install docker-machine from source
+  block:
+  - name: config golang
+    include_role:
+      name: config-golang
+  - name: install docker-machine from source
+    shell: |
+      set -ex
+
+      # Install bats
+      git clone https://github.com/sstephenson/bats.git '{{ ansible_user_dir }}/bats'
+      pushd '{{ ansible_user_dir }}/bats'
+      ./install.sh /usr/local
+      popd
```

Comment:
bats is not related with docker-machine installation, it is only for running tests.

Reply:
OK, will move into job logic